### ISSUE TITLE
the problem ofroast properties

### ISSUE THREAD

**Issue by yunzi-jing** [artisan-roaster-scope/artisan#380]
hello～i am using theartisan-mac-2.0.0.dmg
but boast roast properties is available,click the button display
<class'valueerror’>:
“is not list
how should i solve this problem

**Comment by MAKOMO** [comment]
Could you please provide a full error log or a screenshot of the issue. Please also attach your Artisan settings file (save using menu Help >> Save Settings). Thanks!

**Comment by yunzi-jing** [comment]
Sorry, I don't know how to upload pictures here

**Comment by yunzi-jing** [comment]
May I have your email address? I can send the picture to you by email

**Comment by MAKOMO** [comment]
Just drag and drop it on the edit field. Same for the settings file.

**Comment by yunzi-jing** [comment]
![WechatIMG89](https://user-images.githubusercontent.com/52715412/61480714-240f1680-a9c9-11e9-9533-54520a4c11a3.jpeg)

**Comment by yunzi-jing** [comment]
Oh, thank you ~ this is the text displayed after the properties button in the store

**Comment by yunzi-jing** [comment]
I use version 0.9.7 of the software, this function is available

**Comment by MAKOMO** [comment]
Sorry, we can only support the current version which is v2.0. There is no way to fix anyting in older version. v0.9.7 is already 4 years old!! Please update to the current version and check if the issue persist. Sorry.

**Comment by yunzi-jing** [comment]
The picture I showed you is the problem with the 2.0 version of the software

**Comment by yunzi-jing** [comment]
![WechatIMG96](https://user-images.githubusercontent.com/52715412/61508738-d2dd4200-aa1d-11e9-8197-f3058718677b.jpeg)

**Comment by MAKOMO** [comment]
Hm. Could you indicate in detail how I could reproduce this bug. I do not see a button "Display" in the "Roast Properties" dialog. Could you please also attach your Artisan settings file (again: save via menu Help >> "Save Settings" into an aset file). Thanks!

**Comment by yunzi-jing** [comment]
Thank you, problem solved

**Comment by yunzi-jing** [comment]
I would like to ask, does Artisan have detailed instructions for each function? There are so many things I don't understand how to use

**Comment by MAKOMO** [comment]
Could you please indicate how you solved the issue? The message looked like indicating there is a bug. I would love to fix this for others too. However, I need to be able to reproduce it here and thus I need your instructions on how to do this and also your Artisan settings. Thans!

**Comment by MAKOMO** [comment]
There is online documentation available from the Artisan home page https://artisan-scope.org/. There is also a link to the Artisan blog which contains a lot information and has an Overview page to quickly find the post you are interested in.

**Comment by yunzi-jing** [comment]
Well, the baking curves I saved were saved in version 0.9.7. After opening the old ones with 2.0, the words I gave you in the picture will appear, which cannot be changed. After this problem occurred, I opened a new curve file saved with 2.0 version of the software, and the software returned to normal. This problem was also caused by my improper operation. Thank you for your help ~ this is from China
